include/avif/avif.h

```diff
@@ -176,6 +176,20 @@ typedef enum avifResult
 
 AVIF_API const char * avifResultToString(avifResult result);
 
+// ---------------------------------------------------------------------------
+// avifEncoderHeaderStrategy
+
+typedef enum avifEncoderHeaderStrategy
+{
+    // Encodes as "avif" brand with a MetaBox and all its required boxes for maximum compatibility.
+    AVIF_ENCODER_FULL_HEADER,
+#if defined(AVIF_ENABLE_EXPERIMENTAL_AVIR)
+    // Encodes as "avir" brand with a CondensedImageBox to reduce the encoded file size.
+    // WARNING: Experimental feature. Produces files that are incompatible with older decoders.
+    AVIF_ENCODER_MINIMIZE_HEADER,
```

Comment:
Nit: MINIMIZE => REDUCED ?

"FULL" is an adjective, so it would be good to use an adjective here too.